testutil/keepers/keepers.go

```diff
@@ -1,17 +1,22 @@
 package keepers
 
 import (
+	"fmt"
 	"testing"
 	"time"
 
+	"github.com/cosmos/cosmos-sdk/store"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 
+	storetypes "github.com/cosmos/cosmos-sdk/store/types"
+	bankkeeper "github.com/cosmos/cosmos-sdk/x/bank/keeper"
+	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
 	epochkeeper "github.com/dymensionxyz/dymension-rdk/x/epochs/keeper"
 	mintkeeper "github.com/dymensionxyz/dymension-rdk/x/mint/keeper"
 	seqkeeper "github.com/dymensionxyz/dymension-rdk/x/sequencers/keeper"
 	"github.com/dymensionxyz/rollapp/app"
-
 	tmproto "github.com/tendermint/tendermint/proto/tendermint/types"
+	tmdb "github.com/tendermint/tm-db"
 )
 
 func NewTestEpochKeeperFromApp(t *testing.T, app *app.App) (*epochkeeper.Keeper, sdk.Context) {
```

Comment:
You might want to switch to the new *App that @mtsitrin added in the PR #321

Reply:
I think it isn't merged yet. Should we refactor once it's merged ?